dcos/dcos-ui
# DCOS-50935: TypeError: t.reduce is not a function

Check if port mappins is an array
before attempting to use the reduce function.

Closes https://jira.mesosphere.com/browse/DCOS-50935

## Testing
1. Go to services tab.
2. Open the create service modal.
3. Open the networking tab.
4. Switch network type to "Virtual Network: dcos".
5. Open the JSON editor.
6. Change "portMappings" to something other than an array, for example a string or a number.
7. Verify that no errors are shown in the console.

## Trade-offs
None.

## Dependencies
None.

## Screenshots
### Before
![Peek 2019-08-09 13-31-2](https://user-images.githubusercontent.com/40791275/62773254-63052900-baaa-11e9-81f0-13b1b3998c01.gif)

### After
![Peek 2019-08-09 13-31](https://user-images.githubusercontent.com/40791275/62773266-67314680-baaa-11e9-95ea-6d061105182f.gif)
